.github/workflows/inference-test-partial.yml

```
@@ -0,0 +1,48 @@
+# Tests hitting the inference API are slow and prone to hanging, resulting in
+# timeouts. Therefore, we only run a subset of settings with inference tests by
+# default. See #118
+
+name: inference-tests-partial
+
+on:
+  - push
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  inference-pytest-partial:
+
+    runs-on: ${{ matrix.os }}
+    if: "(github.repository == 'skops-dev/skops') && (!contains(github.event.head_commit.message, '[CI inference]'))"
```

shouldn't this be

```suggestion
    if: ${{ (github.repository == 'skops-dev/skops') && (!contains(github.event.head_commit.message, '[CI inference]')) }}
```

ref: https://docs.github.com/en/actions/learn-github-actions/expressions#example-expression-in-an-if-conditional

---

I'll try that, fingers crossed. If this is really the correct syntax, does this line work correctly:

https://github.com/skops-dev/skops/blob/95a718afd41306c6523d55f110a27b553eb3f7e1/.github/workflows/build-test.yml#L15

?